core/tests/fixtures/utils/order/create.ts

```diff
@@ -0,0 +1,91 @@
+import { faker } from '@faker-js/faker';
+import { z } from 'zod';
+
+const Address = z.object({
+  first_name: z.string(),
+  last_name: z.string(),
+  street_1: z.string(),
+  city: z.string(),
+  state: z.string(),
+  zip: z.string(),
+  country: z.string(),
+});
+
+const OrderResponse = z.object({
+  id: z.number(),
+  customer_id: z.number(),
+  date_created: z.string(),
+  status_id: z.number(),
+  status: z.string(),
+  subtotal_ex_tax: z.string(),
+  subtotal_tax: z.string(),
+  total_inc_tax: z.string(),
+  items_total: z.number(),
+  discount_amount: z.string(),
+  coupon_discount: z.string(),
+  billing_address: Address,
+});
+
+export async function createOrder(productId: number, customerId?: number) {
+  const status_id = 1;
+  const firstName = faker.person.firstName();
+  const lastName = faker.person.lastName();
+  const street1 = faker.location.streetAddress();
+  const city = faker.location.city();
+  const state = faker.location.state();
```

Comment:
🍺 Just to avoid some unknow issues, there's probably way to make sure the state and city are within United States since you're defaulting the country to US.

Reply:
Good catch! Updated to using

`import { faker } from '@faker-js/faker/locale/en_US';`